packages/database/.babelrc.js

```diff
@@ -0,0 +1,38 @@
+const env = {
+  test: {
+    plugins: ['istanbul'],
+  },
+};
+
+const includedDateOffset = 90 * 24 * 60 * 60 * 1000; // include migrations up to 90 days old
+const includedMigrationsDate = new Date().setTime(Date.now() - includedDateOffset);
+const checkMigrationWithinIncludedRange = function (migrationName) {
+  const yearPart = migrationName.substring(0, 4);
+  const monthPart = migrationName.substring(4, 6);
+  const dayPart = migrationName.substring(6, 8);
+  const migrationDate = new Date(`${yearPart}-${monthPart}-${dayPart}`);
+  return migrationDate < includedMigrationsDate;
+};
+
+const ignore = [
+  'src/tests/**',
+  function (filepath) {
+    const filepathComponents = filepath.split('/');
+    const directory =
+      filepathComponents.length > 2 ? filepathComponents[filepathComponents.length - 2] : null;
+    if (directory === 'migrations' || directory === 'migrationData') {
+      const filename = filepathComponents[filepathComponents.length - 1];
+      return checkMigrationWithinIncludedRange(filename);
+    }
+    const parentDirectory =
+      filepathComponents.length > 3 ? filepathComponents[filepathComponents.length - 3] : null;
+    if (parentDirectory === 'migrationData') {
```

Comment:
Hopefully devs will not nest files more than one level in `migrationData`!

Reply:
Ha, yes hopefully!